test/src/test-reader/test-parser.js

```diff
@@ -14,6 +14,7 @@ const proxyquire = require("proxyquire").noCallThru();
 const path = require("path");
```

Comment:
Where are the tests for this module? There should be quite a lot of them here since the main code is in this file.

In `loadFiles` (public) we can check that:
- `config.lastFailed.only` not set -> do not read json files
- `config.lastFailed.only` not set -> do not call addTestFilter (`parse` should be called after `loadFiles`)
- `config.lastFailed.only` set -> read json from one file (string)
- `config.lastFailed.only` set -> read json from one file (string with commas, case with env)
- `config.lastFailed.only` set -> read json from few files (array of strings)
- and other tests

Method `parse` must be tested as well.